### ISSUE TITLE
Add learning rate schedule

### ISSUE THREAD

**Issue by dancivitarese** [jdb78/pytorch-forecasting#1347]
- PyTorch-Forecasting version: pytorch-forecasting | 1.0.0 | pypi_0 | pypi
- PyTorch version: pytorch | 2.0.1 | py3.10_cuda11.8_cudnn8.7.0_0 | pytorch
- Python version: Python 3.10.12 | packaged by conda-forge
- Operating System: Red Hat Enterprise Linux 8

### Expected behavior

I would like to add some PyTorch learning rate schedulers because my TFT model is overfitting really fast.
